packages/azure-services/src/azure-batch/pool-load-generator.spec.ts

```diff
@@ -26,7 +26,7 @@ describe(PoolLoadGenerator, () => {
                     activeToRunningTasksRatio: activeToRunningTasksRatio,
                     addTasksIntervalInSeconds: 15,
                     maxWallClockTimeInHours: 1,
-                };
+                } as JobManagerConfig;
```

Comment:
can you add a value for "sendNotificationTasksCount" instead of casting?, and then check the value in the UTs something like: expect(poolLoadGenerator.sendNotificationTasksCount).toEqual(x);

Reply:
same as above